eq-author-api/constants/pipingAnswerTypes.js

```diff
@@ -1,3 +1,8 @@
-const { TEXTFIELD, NUMBER, CURRENCY, DATE_RANGE } = require("./answerTypes");
+const type = require("./answerTypes");
```

Comment:
Why not destructure the types?

Reply:
Was the bug purely caused by wrong answer types being set here?

Reply:
Yes, it was just the wrong type for piping. It should be `DATE` not `DATE_RANGE`.

I preferred not to destructure just to save typing as you have to state each type twice in this context. But it's just personal preference so I'm happy to change if others prefer the other way.